exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillStatementImpl.java

```diff
@@ -422,6 +507,9 @@ public ResultSet getGeneratedKeys() throws SQLException {
   public int executeUpdate(String sql, int autoGeneratedKeys) throws SQLException {
     throwIfClosed();
     try {
+      if (timeoutTrigger != null) {
```

Comment:
so the trigger is created, but if super.executeUpdate takes more than queryTimeout seconds, the method is not interrupted. I don't believe this is conform to JDBC spec, or somehow useful to the end-user (if query takes 2min but timeout is 60s, exception should be thrown after 60s, not 120s...)